### DISCUSSION TITLE
Git Environments - Ability to Save As and create a Duplicate Environment

### DISCUSSION THREAD

**Discussion by ppmgitrepo** [PrestaShop/PrestaShop#35807] [Ideas]
### The problem

When we define an Environment in Git it can have 30+ secrets and another 20+ Environment variables defined. 
Identical environments need to be created for SIT, UAT, Prod, Dev , Conversions etc.. 

Without the ability to do a Save As and create a copy of the Environment we need to type in everything from scratch which is a tedious process. 
 

### The solution

Please give the ability to select an Environment and do a Save as > Another Environment. 
This way all the Secret variables and Environment variables get created, it is just a matter of changing their values. 

### Alternatives

No alternative at present. 

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by matks** [comment]
Hello @ppmgitrepo are you sure you created that item on the right repository? This is the PrestaShop project and we do not control the GitHub Actions workflow

I think you should rather write your discussion here https://github.com/orgs/community/discussions

**Reply by ppmgitrepo** [reply]
Sorry my bad. 
I moved this to 
https://github.com/PrestaShop/PrestaShop/discussions/35807

**Comment by ppmgitrepo** [comment]
moved to the main community.
